README.adoc

```diff
@@ -7,3 +7,29 @@ Prebuilt images for your Raspberry Pi are available in https://github.com/{gh-na
 Installation procedure is the same as for original Raspbian image which is described https://www.raspberrypi.org/documentation/installation/installing-images/[here].
 
 Images are automatically built on https://travis-ci.org/{gh-name}[Travis CI].
+
+
+== Modifications in this image
+
+Our image is built with the same scripts as the “official” one, with the following modifications:
+
+* Name servers changed to: 8.8.8.8, 217.31.204.130, 2001:4860:4860::8888, 2001:1488:800:400::130.
+* Hostname changed from `raspberrypi` to `hub`.
+* Locale changed from `en_GB.UTF-8` to `en_US.UTF-8`.
+* Keymap changed from `English (UK)` to `English (US) - English (US, international with dead keys)`.
+* Timezone changed from `Etc/UTC` to `Europe/Prague`.
+* Country in `wpa_supplicant.conf` changed from `GB` to `CZ`.
+* OpenSSH daemon enabled by default and removed `AcceptEnv LANG LC_*`.
```

Comment:
Well, ...that is news! 😃 

We will discuss internally how to deal with this matter.